src/NServiceBus.Metrics.PerformanceCounters.Tests/NServiceBus.Metrics.PerformanceCounters.approved.cs

```diff
@@ -25,6 +25,5 @@ public class static PerformanceCountersExtensions
     public class PerformanceCountersSettings
     {
         public void EnableSLAPerformanceCounters(System.TimeSpan sla) { }
-        public void UpdateCounterEvery(System.TimeSpan updateInterval) { }
```

Comment:
We'll need to bump up `NSB.Metrics` dependency, so I don't know if major for both of them would not be better.